### PR TITLE
Documentation improvements: Remove duplicate READMEs, consistent titles

### PR DESCRIPTION
**Pull request by pamelafox**
## Purpose

The Microsoft Learn samples browser now lets us put the YAML inside a comment, so we don't need the duplicate docs anymore. This PR removes them and generally makes the docs more consistent by prefacing each with "RAG chat: ".
This PR also changes the main title to RAG, to match the short title used on other pages.

## Does this introduce a breaking change?

When developers merge from main and run the server, azd up, or azd deploy, will this produce an error?
If you're not sure, try it out on an old environment.

```
[ ] Yes
[X] No
```

## Does this require changes to learn.microsoft.com docs?

This repository is referenced by [this tutorial](https://learn.microsoft.com/azure/developer/python/get-started-app-chat-template)
which includes deployment, settings and usage instructions. If text or screenshot need to change in the tutorial,
check the box below and notify the tutorial author. A Microsoft employee can do this for you if you're an external contributor.

```
[ ] Yes
[X] No
```

## Type of change

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

## Code quality checklist

See [CONTRIBUTING.md](https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/CONTRIBUTING.md#submit-pr) for more details.

N/A

### CODE REVIEW

File: docs/data_ingestion.md [hunk 1]
```diff
@@ -56,6 +53,12 @@ Chunking allows us to limit the amount of information we send to OpenAI due to t
 
 If needed, you can modify the chunking algorithm in `scripts/prepdocslib/textsplitter.py`.
 
+### Enhancing search functionality with data categorization
```
Comment (pamelafox) [inline docs/data_ingestion.md:56]: Moved to match TOC

File: docs/data_ingestion.md [hunk 2]
```diff
@@ -99,3 +102,30 @@ The Azure AI Search indexer will take care of removing those documents from the
 ### Scheduled indexing
 
 If you would like the indexer to run automatically, you can set it up to [run on a schedule](https://learn.microsoft.com/azure/search/search-howto-schedule-indexers).
+
+## Debugging tips
```
Comment (pamelafox) [inline docs/data_ingestion.md:106]: Added per today's conversation

File: docs/data_ingestion.md [hunk 3]
```diff
@@ -1,21 +1,24 @@
-# Indexing documents for the Chat App
+# RAG chat: Data ingestion
```
Comment (pamelafox) [inline docs/data_ingestion.md:1]: I didn't put the frontmatter on this one, as it feels the least like it merits its own page in the samples browser.

File: docs/data_ingestion.md [hunk 4]
```diff
@@ -99,3 +102,30 @@ The Azure AI Search indexer will take care of removing those documents from the
...
+```
+
+To search for specific filenames:
+
+```json
+{
+  "search": "*",
+  "count": true,
+  "top": 1,
+  "filter": "sourcefile eq '209884Orig1s000RiskR.pdf'",
```
Comment (mattgotteiner) [inline docs/data_ingestion.md:128]: Can you change the title of this  PDF please

**Review by mattgotteiner — APPROVED** [review]

File: docs/data_ingestion.md (hunk 4 above)
Comment (pamelafox) [inline docs/data_ingestion.md:128]:
```suggestion
  "filter": "sourcefile eq 'employee_handbook.pdf'",
```

**Pull request merged**